File: Week01/26.Delete_Array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        if (nums.length < 1) return nums.length;
        int j = 0;
        for (int i = 0; i < nums.length; i++) {
            if (nums[i] != nums[j]) {
                j++;
                nums[j] = nums[i];
            }
        }
        return j + 1;
    }
}

File: Week01/26.Delete_Array.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var removeDuplicates = function (nums) {
  if (nums.length < 2) return nums.length
  let i = 1
  while (nums[i] != undefined) {
    if (nums[i] === nums[i - 1]) {
      nums.splice(i, 1)
      continue
    }
    i++
  }
};